app/controllers/posts_controller.rb

```diff
@@ -24,8 +24,11 @@ def metatype_to_contenttype(metatype)
   def find_page(author, title)
     return unless author
 
-    title = title.gsub('-', ' ')
-    author.pages.where('lower(title) = ?', title.downcase).first
+    _title = title.gsub('-', ' ')
+    if !_title.match(/\A([\w]+[\s]*)+\z/)
```

Comment:
The regex pattern should be centralized and not repeated wherever it’s used.

Reply:
@mobitar where do you recommend storing it? `PostsHelper`?

Reply:
I put it in `PostsHelper`, does that sound okay?